### PR TITLE
Implement Potr Pots landing page

### PR DESCRIPTION
**Pull request by anastasiia-tilikina**
[DEMO LINK](https://anastasiia-tilikina.github.io/Potr_Pots/)

### CODE REVIEW

**Comment by qoa11a** [comment]
image
When I submit form I'm redirectered here. Fix it please

**Comment by qoa11a** [comment]
image
Image isn't full width of screen on siome sizes

**Comment by qoa11a** [comment]
image
Some of "Pluses" doesn't work (it seems that blocks with info has 0 opacity and block the "Plus" signs)

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Implement landing page according to [Figma design](https://www.figma.com/file/50
 11. `git push origin develop` - to send you code for PR
 12. Create a Pull Request (PR) from your branch `develop` to branch `master` of original repo.
 13. Replace `<your_account>` with your Github username in the
-  [DEMO LINK](https://<your_account>.github.io/Potr_Pots/)
+  [DEMO LINK](https://anastasiia-tilikina.github.io/Potr_Pots/)
```
Comment (qoa11a) [inline README.md:36]: See comments I've left.

**Review by qoa11a — CHANGES_REQUESTED** [review]

**Comment by anastasiia-tilikina** [comment]
So, I've fixed the form and problem with buttons.
But It seems like with background img everything is fine) It's some strange behavior of dev tools in chrome)
I've tried to delete block by block from landing in order to see which one causes this strange behavior) because it's not only img doesn't take full width but the entire body.
so I turned off responsive mode and tried to resize the window and everything was fine) Then I checked in Safari dev tools and everything looks good as well)

https://user-images.githubusercontent.com/108538921/190176542-2aa16040-884d-476e-a485-936a32a3de7a.mov

**Comment by anastasiia-tilikina** [comment]
And more screenshots
Screenshot 2022-09-14 at 16 58 25
Screenshot 2022-09-14 at 16 50 40
Screenshot 2022-09-14 at 16 49 55

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
this button should not open a new page
image

add hover effects to those images 
image

there is some difference between backgrounds(noticeable line)
image

Screenshot 2022-09-16 at 11 20 46 (2)

Screenshot 2022-09-16 at 11 21 03 (2)

add cursor pointer to submit button
image

2022
image

**Pull request closed without merging**